PIL/TiffTags.py

```diff
@@ -95,8 +95,8 @@ def lookup(tag):
     278: ("RowsPerStrip", LONG, 1),
     279: ("StripByteCounts", LONG, 0),
 
-    280: ("MinSampleValue", LONG, 0),
-    281: ("MaxSampleValue", SHORT, 0),
+    280: ("MinSampleValue", LONG, 1),
+    281: ("MaxSampleValue", SHORT, 1),
```

Comment:
I don't see why this change was made - https://www.awaresystems.be/imaging/tiff/tifftags/minsamplevalue.html and https://www.awaresystems.be/imaging/tiff/tifftags/maxsamplevalue.html